py/desispec/pipeline/db.py

```diff
@@ -279,6 +279,8 @@ def all_tasks(night, nside, expid=None):
                     full["fluxcalib"].append(props)
                     # Add cframe
                     full["cframe"].append(props)
+                    # Add QA
+                    full["qadata"].append(props)
```

Comment:
Ah, on the other hand, this seems to imply that we want "qadata" right after cframes (and before spectra + redshifts).  We made need to do some more creative splicing of the default_task_chain (inserting qadata after the cframe entry).